README.md

```diff
@@ -41,6 +41,7 @@ knowledge_repo --repo ./example_repo create ipynb example_post.ipynb
 5\. Add your post to the repo with path `project/example`
 ```
 knowledge_repo --repo ./example_repo add example_post.ipynb -p project/example
+knowledge_repo --repo ./example_repo add example_post.Rmd -p project/example
```

Comment:
Hi @RMHogervorst  - thanks for adding these! 

Let's change the paths here and below  to e.g. project/rmd_example and project/ipynb_example  so that if someone runs these lines in sequence, they don't conflict.